src/OpenApi/Config/Tags.php

```diff
@@ -25,9 +25,13 @@
     name: Tags::Assets->name,
     description: 'Asset operations to get/update/create/delete assets'
 )]
+#[Tag(
+    name: Tags::Thumbnails->name,
+    description: 'List thumbnails for assets like videos and images'
+)]
 #[Tag(
     name: Tags::Authorization->name,
-    description: 'Login via username and password to get a token or refresh the token'
+    description: 'Login via username and password'
```

Comment:
Should we maybe state that this is now sessions based and you can also logout?

Reply:
Updated now :)